project_PassMeCash_Polkadot.md

```diff
@@ -0,0 +1,164 @@
+| 2. | Modify the User Persona | We will add more User Personas to the current ones as it will affect the PassMeCash-Polkadot. |
+| 3. | Modify the High Fidelity | More pages will be added to the High-Fidelity design. |
+| 4. | Modify the Prototype | Additional pages will be included to the prototype design too. |
+
+
+### Milestone 2 — Development
+
+- **Estimated Duration:** 9 weeks
+- **FTE:**  4
+- **Costs:** 25,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article or Video | We will develop and publish an **article or video** that explains this project in detail for everyone to understand how it works. |
+| 1. | Frontend Development | The front-end would be built using React.js, a popular JavaScript library. This allows for the development of a responsive and user-friendly interface. Tailwind will be used to convert all the Figma files. |
+| 2. | Backend Development | The back-end would be built using Django, a python framework. python would handle the wallet's business logic, such as generating and storing private keys and handling transactions. Through the use of web3.py, we can generate a wallet address for the blockchain and also encrypt it why passing all necessary details to the user via API created with Django |
+| 3. | APIs & Libraries |  Chainlink aggregator would be used to communicate with the Polkadot blockchain nodes and retrieve real time information such as transaction history and account balances. RESTful APIs would be implemented using Django, a python web application framework. 1inch aggregator plugin for swap, Coinbase and Polkadot.js SDKs for Blockchain and token data. |
+| 4. | Smart Contract | Smart contracts would be developed with the implemetation of Ink-OpenBrush, to provide standard contracts based on PSP. Smart contracts would be used to automate certain transaction functions, such as sending and receiving payments. |
+| 5. | Hosting | PassMecash would be hosted on a cloud platform such as Amazon Web Services (AWS) or Google Cloud Platform (GCP). This would ensure high availability and scalability. |
+| 6. | Testing, Debugging & Launch | We will test and debug errors in the codes, thereafter we will deploy and launch. |
```

Comment:
Testing should be part of 0c, and debugging and launch are, without a more thorough specification, not covered by the grants program.

Reply:
The testing, simply mean testing our smart contract on the Polkadot testnet, then if there are errors we debug them. While the launch simply mean to deploy PassMeCash smartcontract to the Polkadot mainnet. All these are key tasks on the PassMeCash - Polkadot project development which will take some man hours. However, it is okay, if it is not covered by the grant.